README.md

```diff
@@ -154,6 +154,7 @@ docker run --rm --network=container:pia alpine:3.10 wget -qO- https://ipinfo.io
 | `BLOCK_NSA` | `off` | `on` or `off`, blocks NSA hostnames |
 | `UNBLOCK` | | comma separated string (i.e. `web.com,web2.ca`) to unblock hostnames |
 | `EXTRA_SUBNETS` | | comma separated subnets allowed in the container firewall (i.e. `192.168.1.0/24,192.168.10.121,10.0.0.5/28`) |
+| `PORT_FORWARDING` | | Set to `true` to read the forwarded port |
```

Comment:
Can you set the default to `false` thanks!

Reply:
It is: https://github.com/qdm12/private-internet-access-docker/blob/55492015cbb62d47d1f97e6067e64f09cd234f18/Dockerfile#L33

Do you mean writing the default value in the Readme ? I think asking to set it to `true` to enable the feature implicitely means it's not enabled by default...

Reply:
Yes but I have a beautiful Default column :smile:

Reply:
Sorry, didn't notice, I'll fix that 😊

Reply:
Done